tests/e2e/transitions/sms_workflows.spec.js

```diff
@@ -115,7 +115,7 @@ describe('SMS workflows', () => {
   
   afterAll(async () => await utils.revertDbNative());
 
-  afterEach(async () => utils.revertDbNative(contacts.map(c => c._id)));
+  afterEach(async () => await utils.revertDbNative(contacts.map(c => c._id)));
```

Comment:
I was going to suggest this:
```suggestion
  afterEach(async () => utils.revertDbNative(contacts.map(c => c._id)));
```

...but I see that's what it was before. Why the change?

Reply:
The eslint plugin we added fails when you leave the await out.  

```
Running "exec:eslint" (exec) task
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

/home/newt/dev/cht-core/tests/e2e/transitions/sms_workflows.spec.js
  118:13  warning  Missing await in async function  async/missing-await-in-async-fn

✖ 1 problem (0 errors, 1 warning)
```

Reply:
Can we change it so it's an error instead of a warning?

Reply:
My point is... most of the time, people won't scroll up to the top of their test output to see if they had lint warnings. So most of the time, they'll go unnoticed.

Reply:
Yeah that's my fault. I didn't look at what the 1 meant and assumed the eslint warning was enough. I have switched it locally to error but I'm waiting on @craig-landry  feedback if we want to stick to the syntax he outlines vs the syntax that doesn't break the plugin rule.

Reply:
I agree with the preference of errors over warnings.

Regarding this one in particular.... :disappointed:  
options seem to be:
1. Leave the code as what I consider "correct" and add a matching "ignore" for the tooling. This is really annoying since those ignore statements are supposed to be more about why some case is a special exemption to an otherwise good rule.
2. Change the code to add the `await` as seen here. Making things worse (in my opinion here) due to a tools constraint is very frustrating. It also means a future reader (probably me again every time I see it) will want to "fix" it.
3. Change the function syntax to include a `return`. I am curious if that makes the tooling evaluate it differently. If it does, this is technically better than option 2, since it has no actual impact to what is happening, it's just an alternative syntax that is equally correct, but just less optimal.
4. Maybe I just have this all wrong and the tool is saving me from my ignorance. This is actually the best outcome because then we can feel comfortable with the tool and the universe is safe.

In the end here, I'm hoping for 4. If that's not it, I can calm down and let this go as 2.